src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -41,6 +41,11 @@ class InstPrepdisk
       def run
         return :auto if Mode.update
 
+        storage = manager.storage
+
+        storage.probed.save(Directory.logdir + "/probed-devicegraph.xml")
+        storage.staging.save(Directory.logdir + "/staging-devicegraph.xml")
+
```

Comment:
I have tested the code and it works.

Since the XML files can be very huge I prefer to have them as separate files. It must also be easy to load them on another system without first cutting them from the logs to create test cases.

Whether the code is here or in the manager I do not mind.